tests/admin_scripts/tests.py

```diff
@@ -2879,6 +2895,26 @@ def test_invalid_name(self):
                 )
                 self.assertFalse(os.path.exists(testproject_dir))
 
+    def test_output(self):
+        dirs = {
+            "foo": self.test_dir,
+            # Use a subdirectory so it is outside the PYTHONPATH.
+            "bar": (
+                os.path.join(self.test_dir, "apps/otherdir")
+                if os.name == "posix"
+                else os.path.join(self.test_dir, "apps\\otherdir")
```

Comment:
What do you think about using `pathlib` rather than `os`? For example, this part could be rewritten as:

```suggestion
                "bar": Path(self.test_dir) / "apps/otherdir",
```